LayoutControl.tis

```diff
@@ -1,15 +1,15 @@
-class LayoutControl
+class LayoutController
```

Comment:
the commit should be more atomised
If you named it as '.... change through events' it should contain exactly that change.
Other refactoring should be committed separately